libtiledbsoma/include/tiledbsoma/spdlog_with_R.h

```diff
@@ -11,8 +11,8 @@
 
 // define R_R_H and USING_R so that REprintf and R_FlushConsole are used
 // by spdlog, and also declare them hereby
-#define R_R_H
-#define USING_R
+//#define R_R_H
+//#define USING_R
```

Comment:
Can we make this conditional, and/or you just exclude this header file from your Python build?